### ISSUE TITLE
Relational models: class names are incorrect.

### ISSUE THREAD

**Issue by kddige** [kimjbstar/prisma-class-generator#30]
## Expected Behavior
```ts
import { License_types } from './license_types'
import { Media_type } from '@prisma/client'

export class Media {
	type?: Media_type = Media_type.other

	name?: string = undefined

	url?: string = undefined

	license_types: License_types[] = undefined
}
```
## Actual Behavior
```ts
import { LicenseTypes } from './license_types'
import { Media_type } from '@prisma/client'

export class Media {
	type?: Media_type = Media_type.other

	name?: string = undefined

	url?: string = undefined

	license_types: License_types[] = undefined
}
```

## Steps to Reproduce the Problem

1. Create 2 Prisma models with underscore: "some_table_name".
2. Add relational field in one of the models. Dosen't matter what kind.
3. Run ```npx prisma generate```

## Specifications

-   Version: 0.2.6
-   Prisma Version: 4.13.0
-   Platform: aarch64 (darwin)

**Comment by rtroberts** [comment]
I opened PR #37 which should fix the inconsistency in this case.

**Comment by kddige** [comment]
Resolved by #37
